src/main/java/com/altenheim/kalender/controller/viewController/SettingsViewController.java

```diff
@@ -1,20 +1,81 @@
 package com.altenheim.kalender.controller.viewController;
 
 import com.altenheim.kalender.models.SettingsModel;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.MenuButton;
+import javafx.scene.control.MenuItem;
+import javafx.scene.control.RadioButton;
+import javafx.scene.control.TextField;
+import javafx.scene.text.Text;
 
 public class SettingsViewController extends ResponsiveController
 { 
     private SettingsModel settings;
 
+    @FXML
+    private MenuButton selectionSpecialField, selectionCourse, selectionSemester, btnMenuScrapingIntervall;
+    @FXML
+    private Button btnImport, btnExport, btnSave;
+    @FXML
+    private TextField inputStreet, inputCity, inputZipCode, inputHouseNumber, inputMail;
+    @FXML
+    private RadioButton adressMailFirst, toolTipsOn, toolTipsOff;
+    @FXML
+    private Text scrappingURL, adressTitle, showStreet, showHouseNumber, showCity, showZipCode, showMail;
+    @FXML
+    private MenuItem selectionSpecialFieldInsurance, selectionSpecialFieldWi;
+    
+    
     public SettingsViewController(SettingsModel settings)
     {
         this.settings = settings;
     }
     
-    public void changeContentPosition() 
+    @FXML
+    private void initialize ()
     {
-        
-    }    
-}
-
-
+        showStreet.textProperty().bind(settings.getStreet());
+        showHouseNumber.textProperty().bind(settings.getHouseNumber());
+        showZipCode.textProperty().bind(settings.getZipCOde());
+        showCity.textProperty().bind(settings.getCity());
+        showMail.textProperty().bind(settings.getMail());
+        //entfernen sobald es funktioniert
+        scrappingURL.setText(settings.getCalendarParser());
+    }
+    
+    @FXML 
+    void selectionScrapper(ActionEvent event)
+    {
```

Comment:
Schon näher dran. Aber den Cast und get Source musste natürlich nicht jedes mal in der Abfrage machen. Einmal am Anfang und dann im if einfach Objektvergleich.